src/crab_verifier.hpp

```
@@ -29,25 +29,25 @@ class Report final {
-        for (const auto& [label, reach_warning] : reachability) {
-            for (const auto& msg : reach_warning) {
+        for (const auto& [label, reach_error] : reachability) {
+            for (const auto& msg : reach_error) {
                 result.insert(to_string(label) + ": " + msg);
             }
         }
         return result;
     }
 
-    std::set<std::string> warning_set() const {
+    std::set<std::string> error_set() const {
```

Clang (14.0.6)

[warning] function 'error_set' should be marked [[nodiscard]]

(modernize-use-nodiscard)